pkgs/os-specific/darwin/apple-sdks/frameworks/11.3.0/fixups.nix

```diff
@@ -0,0 +1,102 @@
+# TODO(@connorbaker): Copied from 11.0 -- should be regenerated.
```

Comment:
I'd prefer if we had a single declaration which handles version differences internally.

DRY.